### PR TITLE
[Feature] Stop notifying updates which have an older semver release

### PR DESCRIPTION
**Pull request by RodoMa92**
Please tick as appropriate:
- [ ] I have tested this code on a steam deck or on a PC
- [x] My changes generate no new errors/warnings
- [ ] This is a bugfix/hotfix
- [x] This is a new feature

# Description
I was annoyed from the constant notification from older testing versions if they were still up. So this will filter notification following the standard semver numbering scheme. Needs testing.

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -52,6 +52,8 @@
     "react-i18next": "^12.3.1",
     "react-icons": "^4.9.0",
     "react-markdown": "^8.0.7",
-    "remark-gfm": "^3.0.1"
+    "remark-gfm": "^3.0.1",
+    "compare-versions": "^6.1.0",
+    "semver": "^7.6.0"
```
[the remark this answers is not in the document]
Comment (RodoMa92) [reply frontend/package.json:57]: Whoops, was testing different libraries, it's not needed. I'll drop it shortly.

File: package-lock.json
[the remark this answers is not in the document]
Comment (RodoMa92) [reply package-lock.json:1]: Yep, deleting it now.

**Review by JohnnyCrazy — APPROVED** [review]

**Review by JohnnyCrazy — APPROVED** [review]

**Comment by TrainDoctor** [comment]
@RodoMa92 if you can rebase this onto latest main I'd love to give this a test and see if it's working now.

**Comment by RodoMa92** [comment]
Assuming I haven't effed up the huge merge, this should be rebased now.

**Comment by TrainDoctor** [comment]
@SteamDeckHomebrew/decky-testers please give this a whirl.

**Comment by RodoMa92** [comment]
Same for this, I'll rebase it shortly.

**Pull request closed without merging**